docbook/README.md

```diff
@@ -0,0 +1,25 @@
+
+# Sources of the documentation are written in the DocBook format
+
+It builds automatically when you push your changes to the GitHub,
+but you may want to check how it looks before pushing changes.
+
+To check how it looks without pushing changes you have to prepare your
+machine to build DocBook 5.1.
+
+## Build
+
+If you are new to DocBook and XLST precessor check out good introductory video
+["Installing and Using Saxon for your XSLT Development" by Peter K. Johnson](https://youtu.be/FsDq2-VV0Uo)
+
+### Prerequisites
+
+* [Java 8](https://www.java.com/en/download/)
+* [Saxon-PE](http://www.saxonica.com/download/java.xml)
```

Comment:
@DavidFatDavidF you've mentioned in https://github.com/GALAglobal/TAPICC/issues/3#issuecomment-357941694:
> So far we are using only preliminary, built-in oXygen transforms but Jano is working on modified stylesheets for HTML and PDF

Can you provide direct links to the missing prerequisites, please?